### PR TITLE
MM-23419: capture output from ssh commands

### PR DESCRIPTION
**Pull request by agnivade**
#### Summary

capture output from ssh commands

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-23419

### CODE REVIEW

File: deployment/terraform/ssh/ssh.go [hunk 1]
```diff
@@ -71,43 +71,51 @@ func (ea *ExtAgent) NewClient(ip string) (*Client, error) {
...
+	sess.Stderr = &out
 
-	return sess.Run(cmd)
+	err = sess.Run(cmd)
+	return out.String(), err
 }
 
 // Upload uploads a given src object to a given destination file.
-func (sshc *Client) Upload(src io.Reader, dst string, sudo bool) error {
+func (sshc *Client) Upload(src io.Reader, dst string, sudo bool) (string, error) {
```
Comment (isacikgoz) [inline deployment/terraform/ssh/ssh.go]: Nit: To be more consistent with similar methods like `cmd.Output()`, maybe we can return `[]byte`.

Comment (agnivade) [reply deployment/terraform/ssh/ssh.go]: Sure.

**Comment by agnivade** [comment]
Going to wait on this until the agent cluster PR is also in. And then make all the changes and submit again.

**Comment by streamer45** [comment]
I think this can move forward now, correct @agnivade ?

**Comment by agnivade** [comment]
Yep, I will get to this.

**Pull request merged**